src/OrchardCore.Modules/OrchardCore.Media/Controllers/MediaCacheController.cs

```diff
@@ -19,13 +22,13 @@ public class MediaCacheController : Controller
 
         public MediaCacheController(
             IAuthorizationService authorizationService,
-            IMediaFileStoreCache mediaFileStoreCache,
+            IServiceProvider serviceProvider,
             INotifier notifier,
             IHtmlLocalizer<MediaCacheController> htmlLocalizer
             )
         {
             _authorizationService = authorizationService;
-            _mediaFileStoreCache = mediaFileStoreCache;
+            _mediaFileStoreCache = serviceProvider.GetService<IMediaFileStoreCache>();
```

Comment:
I understand why. But please add a comment for when I have lost my mind, and the other devs that would read this code.

Reply:
🤣 Done